zip-0316.rst

```diff
@@ -431,7 +431,7 @@ The following FVK or IVK Encodings are used in place of the
 
 * A Sapling IVK Encoding, also with Typecode :math:`\mathtt{0x02},`
   is an encoding of :math:`(\mathsf{dk}, \mathsf{ivk})` given by
-  :math:`\mathsf{I2LEOSP}_{88}(\mathsf{dk})\,||\,\mathsf{I2LEOSP}_{256}(\mathsf{ivk}).`
+  :math:`\mathsf{I2LEOSP}_{256}(\mathsf{dk})\,||\,\mathsf{I2LEOSP}_{256}(\mathsf{ivk}).`
```

Comment:
In fact, I wonder if the function around `dk` should drop out entirely, because 5.6.4.3 of the zcash protocol states it as just `dk`: 
![image](https://github.com/zcash/zips/assets/3548/e5fe7e25-335b-4517-b733-0aef887a94ec)